tests/test_plugins.py

```diff
@@ -47,6 +47,23 @@ def test_article_list_plugin_article_count():
 
 
 @pytest.mark.django_db
+def test_article_list_plugin_show_flag():
+    """
+    Test article list plugin showing only the articles with the
+    show_in_article_list_plugin flag set to True
+    """
+    create_articles(3)
+    hidden_article = Article.objects.get(pk=3)
```

Comment:
Use `last()` instead of `get(pk=3)`

Reply:
thought so, but IIRC `last()` was quite new feature so chose to use more backward-compatible instead. This PR is probably obsolete now anyway but thanks for the review.